server/models/operationtype.coffee

```diff
@@ -0,0 +1,38 @@
+americano = require 'americano'
+hashmap = require 'hashmap'
+module.exports = OperationType = americano.getModel 'operationtype',
+    name: String
+    weboobValue: Number
+
+ListOperationType = new hashmap()
```

Comment:
I think you can use a plain old JS object here, to avoid the dependency on `hashmap`.

```
var map = new hashmap(); // will become: var map = {};
map.set(key, value); // will become: map[key] = value;
map.get(key); // will become : map[key]
if map.get(key)? {} // will become if (key in map) {}
```